aiidalab_widgets_base/viewers.py

```diff
@@ -383,10 +397,14 @@ def _cell_tab(self):
         self.cell_beta = ipw.HTML()
         self.cell_gamma = ipw.HTML()
 
-        self._observe_cell()
+        self.cell_spacegroup = ipw.HTML()
+        self.cell_hall = ipw.HTML()
+
+        self._observe_ase_structure()
 
         return ipw.VBox(
             [
+                ipw.HTML("Length unit: Angstrom"),
```

Comment:
Afaik the unit, unlike the name, is typically not capitalized.
```suggestion
                ipw.HTML("Length unit: angstrom"),
```